varcode/variant.py

```diff
@@ -50,18 +50,37 @@
     ExonLoss,
     ExonicSpliceSite,
 )
-from .nucleotides import normalize_nucleotide_string
+from .nucleotides import normalize_nucleotide_string, STANDARD_NUCLEOTIDES
 from .string_helpers import trim_shared_flanking_strings
 from .transcript_helpers import interval_offset_on_transcript
 
 class Variant(object):
+    __slots__ = (
+        "contig",
+        "start",
+        "end",
+        "ref",
+        "alt",
+        "ensembl",
+        "original_ref",
+        "original_alt",
+        "original_start",
+        "_short_description",
```

Comment:
Yeah, it's for memory. It makes it possible to load in dbsnp on my machine. Whether that is a use case we care about I suppose might be debatable. Since the errors that are raised if this gets out of sync with the Variant attributes will be very obvious, I think it's worth keeping for now, despite the maintenance hassle.